elytron/src/test/java/org/wildfly/extension/elytron/RealmsTestCase.java

```diff
@@ -246,6 +254,87 @@ public void testFileSystemRealmEncodingAndCharset() throws Exception {
 
     }
 
+    /**
+     * Test the filesystem realm can handle identities with different types of encrypted passwords and attributes. Also
+     * ensures a pre-assigned SecretKey as well as a newly generated SecretKey work.
+     */
+    @Test
+    public void testFilesystemRealmEncrypted() throws Exception {
+        KernelServices services = super.createKernelServicesBuilder(new TestEnvironment()).setSubsystemXmlResource("realms-test.xml").build();
+        if (!services.isSuccessfulBoot()) {
+            Assert.fail(services.getBootError().toString());
+        }
+
+        ServiceName serviceName = Capabilities.SECURITY_REALM_RUNTIME_CAPABILITY.getCapabilityServiceName("FilesystemRealm5");
+        ModifiableSecurityRealm securityRealm = (ModifiableSecurityRealm) services.getContainer().getService(serviceName).getValue();
+        ServiceName serviceNameGeneratedKey = Capabilities.SECURITY_REALM_RUNTIME_CAPABILITY.getCapabilityServiceName("FilesystemRealm6");
+        ModifiableSecurityRealm securityRealmGeneratedKey = (ModifiableSecurityRealm) services.getContainer().getService(serviceName_randomKey).getValue();
```

Comment:
Thanks for updating the variable names to camel case. Was just building locally to do some test runs and noticed there are a few more places in this file where the variable names still have an underscore, resulting in compilation issues.